sdk/devcenter/azure-developer-devcenter/src/samples/java/com/azure/developer/devcenter/ReadmeSamples.java

```diff
@@ -45,53 +48,60 @@ public void readmeSamples() {
             System.out.println(p);
         }
         // END: com.azure.developer.devcenter.readme.getProjectsAndPools
-        
+
         // BEGIN: com.azure.developer.devcenter.readme.createAndConnectToDevBox
         // Provision a Dev Box
         BinaryData devBoxBody = BinaryData.fromString("{\"poolName\":\"MyPool\"}");
         SyncPoller<BinaryData, BinaryData> devBoxCreateResponse =
                 devBoxClient.beginCreateDevBox("myProject", "me", "MyDevBox", devBoxBody, null);
         devBoxCreateResponse.waitForCompletion();
 
-        
+
         Response<BinaryData> remoteConnectionResponse =
                         devBoxClient.getRemoteConnectionWithResponse("myProject", "me", "MyDevBox", null);
         System.out.println(remoteConnectionResponse.getValue());
         // END: com.azure.developer.devcenter.readme.createAndConnectToDevBox
-        
+
         // BEGIN: com.azure.developer.devcenter.readme.deleteDevBox
         // Tear down the Dev Box when we're finished:
         SyncPoller<BinaryData, Void> devBoxDeleteResponse =
                         devBoxClient.beginDeleteDevBox("myProject", "me", "MyDevBox", null);
-        devBoxDeleteResponse.waitForCompletion();        
+        devBoxDeleteResponse.waitForCompletion();
         // END: com.azure.developer.devcenter.readme.deleteDevBox
         // END: com.azure.developer.devcenter.readme.devboxes
 
         // BEGIN: com.azure.developer.devcenter.readme.environments
         // BEGIN: com.azure.developer.devcenter.readme.createEnvironmentsClient
-        EnvironmentsClient environmentsClient =
-                        new EnvironmentsClientBuilder()
-                                .endpoint(endpoint)
-                                .credential(new DefaultAzureCredentialBuilder().build())
-                                .buildClient();
+        DeploymentEnvironmentsClientBuilder deploymentEnvironmentsClientbuilder =
+                        new DeploymentEnvironmentsClientBuilder()
+                            .endpoint("ENDPOINT")
+                            .httpClient(HttpClient.createDefault())
+                            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC));
```

Comment:
Are the `httpClient` and `httpLogOptions` necessary for the sample? Again, seems like a delta from the dev box equivalent

Reply:
I think we need creds in all of our clients. So, I added those and removed the log options.

Reply:
Fixed in: https://github.com/Azure/azure-sdk-for-java/pull/34918/commits/9c48ce254c15edd43a7950c786d27b11524769a9